src/components/functions/index.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import React from 'react';
+import { RunningTimesByFunc } from './RunningTimesByFunc';
+import { FunctionCallers, FunctionCallees } from './FunctionButterfly';
+import type { IndexIntoFuncTable } from '../../types/profile';
+
+import './FunctionsPanel.css';
+
+type State = {|
+  selectedFuncIndex: IndexIntoFuncTable | null,
+|};
```

Comment:
This may be stored in a redux state later, so that other parts of the app can also react to this selection.